elife/config/etc-init.d-goaws

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          goawsdaemon
+# Required-Start:    $local_fs $network $syslog
+# Required-Stop:     $local_fs $network $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: GoAWS
+# Description:       Start/stop daemon for GoAWS.
+### END INIT INFO
+
+NAME="goaws"
+PIDFILE="/var/run/goaws/goaws.pid"
+PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
+APPDIR="/"
+APPBIN="/usr/local/bin/goaws"
+
+# Include functions
+set -e
+. /lib/lsb/init-functions
+
+start() {
+  printf "Starting '$NAME'... "
+  start-stop-daemon --start --background -m --oknodo --pidfile ${PIDFILE} --make-pidfile --chuid root:root --exec ${APPBIN}
+  aws sns create-topic --name=annual-reports --endpoint=http://localhost:4100
+  aws sns create-topic --name=blog-articles --endpoint=http://localhost:4100
+  aws sns create-topic --name=collections --endpoint=http://localhost:4100
+  aws sns create-topic --name=covers --endpoint=http://localhost:4100
+  aws sns create-topic --name=events --endpoint=http://localhost:4100
+  aws sns create-topic --name=interviews --endpoint=http://localhost:4100
+  aws sns create-topic --name=labs-experiments --endpoint=http://localhost:4100
+  aws sns create-topic --name=people --endpoint=http://localhost:4100
+  aws sns create-topic --name=podcast-episodes --endpoint=http://localhost:4100
+  aws sns create-topic --name=press-packages --endpoint=http://localhost:4100
+  aws sns create-topic --name=subjects --endpoint=http://localhost:4100
+  aws sqs create-queue --queue-name journal-cms--queue-local --endpoint=http://localhost:4100
```

Comment:
If you're just going to customize it with the environment name (which is the best solution for consistency reasons) you can refer to `pillar.elife.env`.

If instead you want wildly different values, just refer to `pillar.elife.goaws.something` and provide a sample value in `pillar/elife.sls` (which will be used locally). Then list the values you want and we'll add it to the private `builder-private` repository setting them for each environment. Essentially we already customize the pillars per each environment, so that Salt `.sls` file just needs to refer to a single pillar value.